Gems/PhysX/Code/Tests/PhysXMeshColliderComponentModeTests.cpp

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <TestMeshColliderComponent.h>
+
+#include <AzManipulatorTestFramework/AzManipulatorTestFrameworkTestHelpers.h>
+#include <AzToolsFramework/UnitTest/AzToolsFrameworkTestHelpers.h>
+#include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+
+namespace UnitTest
+{
+    class PhysXMeshColliderComponentModeTest
+        : public ToolsApplicationFixture<false>
+    {
+    protected:
+        using EntityPtr = AZ::Entity*;
+
+        AZ::ComponentId m_meshColliderComponentId;
+
+        EntityPtr CreateMeshColliderComponent()
```

Comment:
I find the name of this function quite confusing. It's a bit verbose, but something like CreateEntityWithTestMeshColliderComponent would make more sense to me.

Reply:
Same, it comes from `PhysXColliderComponentModeTest` which has `CreateColliderComponent`. But your name is much better, I'll rename them in both.